app/data_model/submitted_responses.py

```diff
@@ -0,0 +1,61 @@
+import boto3
+from botocore.exceptions import BotoCoreError, ConnectionError as BotoConnectionError
+from botocore.config import Config
+from app.settings import EQ_ENVIRONMENT_PREFIX, EQ_DYNAMODB_ENDPOINT
+from structlog import get_logger
+
+logger = get_logger()
+
+
+class SubmittedResponses(object):
```

Comment:
I think we should create this once for the application maybe in `setup.py` and then reuse it?

Reply:
Done.